From 1def5b4278d97f197520d23c1dce52f93a1b2f46 Mon Sep 17 00:00:00 2001
From: Andrej Valek <andrej.valek@siemens.com>
Date: Tue, 9 Apr 2019 09:40:30 +0200
Subject: [PATCH] splay.cc: fix bind is not a member of std

fix
| ../../squid-4.6/test-suite/splay.cc:134:28: error: 'bind' is not a member of 'std'
|      auto nextRandom = std::bind (distribution, generator);
|                             ^~~~
| ../../squid-4.6/test-suite/splay.cc:134:28: note: 'std::bind' is defined in header '<functional>'; did you forget to '#include <functional>'?

Signed-off-by: Andrej Valek <andrej.valek@siemens.com>
---
 test-suite/splay.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test-suite/splay.cc b/test-suite/splay.cc
index f71b337..4e21adc 100644
--- a/test-suite/splay.cc
+++ b/test-suite/splay.cc
@@ -20,6 +20,7 @@
 #include <unistd.h>
 #endif
 #include <random>
+#include <functional>
 
 class intnode
 {
-- 
2.11.0

